Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "fastlane"
+gem "fastlane", "2.171.0" # further versions break Danger
```

Comment:
In what way? Do we have a ticket to track that?

Reply:
It's really odd issue. It looks like something is modifying Dangerfile by adding a key that Danger does not recognize. The issue seems to be very fresh (after yesterday's release of fastlane) and specific (because danger-xcov is not a popular plugin). `Invalid `Dangerfile` file: Could not find option 'skip_package_dependencies_resolution'`. I think that we could create an issue ticket on danger-xcov github page. What do you think?

Reply:
Sure, please create an issue on the danger-xcov github and reference it here and ideally in your Gemfile comment too. Thanks.

Reply:
Ok done.
https://github.com/fastlane-community/danger-xcov/issues/35